nil/services/rpc/rawapi/pb/conversion.go

```diff
@@ -1370,3 +1370,37 @@ func (r *SendTransactionRequest) PackProtoMessage(transactionSSZ []byte) error {
 func (r *SendTransactionRequest) UnpackProtoMessage() ([]byte, error) {
 	return r.TransactionSSZ, nil
 }
+
+func (txn *RawTxnsResponse) PackProtoMessage(txnWithHash []*types.Transaction, err error) error {
```

Comment:
```suggestion
func (txn *RawTxnsResponse) PackProtoMessage(txn []*types.Transaction, err error) error {
```

This type doesn't include hash